docs/maintainer-guide/issues.md

```diff
@@ -0,0 +1,58 @@
+    * Label requests for changes to existing features with the "enhancement" and "proposal" labels
+    * Label requests for completely new features with the "feature" and "proposal" labels
+    * Use an appropriate label for the part of the project the issue refers to
+1. If it's a bug, be sure the issue includes the following (ask for any missing information):
+    * The ESLint version being used
+    * The configuration being used and the actual source that reproduces the problem
+    * The actual ESLint output (not a summary, have them copy-paste it)
+    * What they expected to happen
+1. If it's an enhancement or feature request, be sure the issue mentions the primary use case (the problem they are trying to solve) and not just a suggestion for a technical change (which needs to be a solution to a problem). If the use case isn't clear, ask for more information. Frequently, "What problem are you trying to solve?" can clarify that.
+1. If it's just a question (one that would normally go to the mailing list or chatroom), feel free to answer the question and close the issue when the conversation is over.
+1. If you can verify that the issue is a duplicate of another issue, mention this in a comment and close the issue. Be sure to reference the issue number of the issue they should refer to, such as, "Duplicate of #1234."
+1. Regardless of the above, always leave a comment. Don't just add labels, engage with the person who opened the issue by asking a question (request more information if necessary) or stating your opinion of the issue. If it's a verified bug, ask if the user would like to submit a pull request.
+
+**Note:** Don't add an "accepted" label to an issue unless it's a bug that you've been able to reproduce and verify (you're sure it's a bug). The "accepted" label will be added by a project admin if it's appropriate for the roadmap.
+
+## Evaluating an Issue
+
+If you're a project admin, then you'll need to evaluate incoming requests for inclusion in the formal roadmap. You should label an issue as "accepted" when all of the following are true:
+
+1. The feature or enhancement is in scope for the project and should be added to the roadmap
+1. You are committed to including the change within the next year
+1. You are reasonably certain of who will do the work
+
+When a suggestion is too ambitious or would take too much time to complete, it's better not to accept the proposal. Stick to small, incremental changes and lay out a roadmap of where you'd like the project to go eventually. Don't let the project get bogged down in big features that will take a long time to complete.
+
+## When to Close an Issue
+
+There are several times when it's appropriate to close an issue:
+
+1. The issue is a duplicate of an existing issue
+1. The issue is just a question rather than a task to be completed
+1. The issue has been open for a year or more with no sign of progress
```

Comment:
@ilyavolodin I tend to use a year because it's hard to argue with. "Really, it's been a year and no movement?"

@lo1tuma if something is that important then it should have somehow bubbled up within a year. Anything that hangs around this long can't possibly be important enough to keep open.